contributors/triage/guideline.md

```diff
@@ -0,0 +1,60 @@
+
+## Scope
+
+This guideline serves as a primary document for triaging incoming issues to Kubernetes. SIGs and projects are encouraged to either use this guideline, or use this as a starting point if necessary. For example if your SIG has specific triaging needs, extend this guideline. 
+
+## Using the bot
+
+Most people do not have the ability to interact with the github issues and labels, for that we use a bot to manage labelling and triaging. The bot has a set of [commands and permissions](https://github.com/kubernetes/test-infra/blob/master/commands.md), this document will cover the basic ones. 
+
+
+## Determine if it’s an issue or support request
+
+Sometimes users ask for support requests in issues; these are usually support requests for people who need help configuring some aspect of Kubernetes. These should be directed to our [support structures](https://github.com/kubernetes/community/blob/master/contributors/devel/on-call-user-support.md) and then closed. Only authors and assignees can `/close` an issue. If the bug is clearly abandoned or in the wrong place you can `/assign` to have it assigned to yourself, and then `/close`.
+
+## Validate if the issue is real
+
+Validate if the problem is real by reproducing it. If real, move to the next step of finding a right SIG. You may need to contact the issue reporter in the following cases: 
+* Do a quick duplicate search to see if the issue has been reported already. If duplicate, let the issue reporter know it by marking it duplicate.
+* If you can not reproduce the issue, contact issue reporter with your findings and close the issue if both the parties agree that it could not be reproduced.
+* If the issue is non-trivial to reproduce work with issue reporter or move to next step of finding a right SIG, and let SIG know of your findings.
+* If you do not get response in 15 days, then close the issue with appropriate comment.
+
+## Find the right SIG(s)
+Components are divided among[ Special Interest Groups (SIGs)](https://github.com/kubernetes/community/blob/master/sig-list.md). Find a proper SIG for the ownership of the issue using the bot:
+
+* Typing `/sig network` in a comment should add the sig/network label, for example.
+* Multiword SIGs use dashes, for example `/sig cluster-lifecycle`.
+
+## Define priority
+After you validate the issue, optionally, per your understanding add a comment of what you think of priority of fixing issue - critical, high, medium, low, wishlist. This doesn’t required to be accurate. The owner from appropriate SIG will assign a correct priority. 
+* `priority/failing-test or priority/critical-urgent` (P0 - Critical - fix it now, it’s a blocker)
```

Comment:
nit: perhaps take the `or` outside of the backticks
e.g.: `priority/failing-test` or `priority/critical-urgent`